.githooks/pre-push

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env ruby
+
+puts "Attempting to push..."
+
+lines = `git fetch origin main && git diff --name-only origin/main | grep "\.rbi$"`.lines
```

Comment:
This won't work if there is a change in `index.json` only.

Reply:
For example I pushed this and the hooks didn't catch anything: https://github.com/Shopify/rbi-central/compare/at-test?expand=1